--- a/drivers/net/wireless/realtek/rtl8852bs/include/rtw_debug.h
+++ b/drivers/net/wireless/realtek/rtl8852bs/include/rtw_debug.h
@@ -40,6 +40,17 @@
 #define _RTW_INFO(x,...) do {} while (0)
 #define _RTW_DBG(x,...) do {} while (0)
 #define _RTW_PRINT_SEL(x,...) do {} while (0)
 
+#define RTW_PRINT_LMT(x, ...) do {} while (0)
+#define RTW_ERR_LMT(x, ...) do {} while (0)
+#define RTW_WARN_LMT(x,...) do {} while (0)
+#define RTW_INFO_LMT(x,...) do {} while (0)
+#define RTW_DBG_LMT(x,...) do {} while (0)
+#define _RTW_PRINT_LMT(x, ...) do {} while (0)
+#define _RTW_ERR_LMT(x, ...) do {} while (0)
+#define _RTW_WARN_LMT(x,...) do {} while (0)
+#define _RTW_INFO_LMT(x,...) do {} while (0)
+#define _RTW_DBG_LMT(x,...) do {} while (0)
+
 #define RTW_INFO_DUMP(_TitleString, _HexData, _HexDataLen) do {} while (0)
 #define RTW_DBG_DUMP(_TitleString, _HexData, _HexDataLen) do {} while (0)
 #define RTW_PRINT_DUMP(_TitleString, _HexData, _HexDataLen) do {} while (0)
--- a/drivers/net/wireless/realtek/rtl8852bs/os_dep/linux/wifi_regd.c
+++ b/drivers/net/wireless/realtek/rtl8852bs/os_dep/linux/wifi_regd.c
@@ -1276,7 +1276,8 @@ int rtw_regd_init(struct wiphy *wiphy)
 	wiphy->regulatory_flags &= ~REGULATORY_DISABLE_BEACON_HINTS;
 #endif
 
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 19, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 19, 0)) && \
+	(LINUX_VERSION_CODE < KERNEL_VERSION(6, 1, 0))
 	wiphy->regulatory_flags |= REGULATORY_IGNORE_STALE_KICKOFF;
 #endif
 
